cross/argtable/Makefile

```diff
@@ -0,0 +1,16 @@
+PKG_NAME = argtable
+PKG_VERS = 2.13
+PKG_EXT = tar.gz
+PKG_DIR = $(PKG_NAME)$.2-13
```

Comment:
Weird...I thought I added a comment here, but maybe I forgot to add it. This line should be changed. I'd suggest to replace your L1-L6 with the following:
```
PKG_NAME = argtable
PKG_VERS_MAJOR = 2
PKG_VERS_MINOR = 13
PKG_EXT = tar.gz
PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS_MAJOR)-$(PKG_VERS_MINOR).$(PKG_EXT)
PKG_DIST_SITE = http://sourceforge.net/projects/$(PKG_NAME)/files/$(PKG_NAME)/$(PKG_NAME)-$(PKG_VERS_MAJOR).$(PKG_VERS_MINOR)
PKG_DIR = $(PKG_NAME)$(PKG_VERS_MAJOR)-$(PKG_VERS_MINOR)
```